code/game/objects/items/weapons/grenades/syndieminibomb.dm

```diff
@@ -46,7 +46,7 @@
 		if(isfloorturf(T))
 			var/turf/open/floor/F = T
 			F.wet = TURF_WET_PERMAFROST
-			addtimer(CALLBACK(F, /turf/open/floor.proc/MakeDry), rand(3000, 3100), TURF_WET_PERMAFROST)
+			addtimer(CALLBACK(F, /turf/open/floor.proc/MakeDry, TURF_WET_PERMAFROST), rand(3000, 3100))

```

Comment:
Make a PR that JUST has the good changes like this.

So you know, it doesn't get delayed while we argue over the addtimer change.